ostelco-ios-client/ViewControllers/ESim/ESIMPendingDownloadViewController.swift

```diff
@@ -113,4 +111,19 @@ class ESIMPendingDownloadViewController: UIViewController {
             }
         }
     }
+    
+    override func viewDidLoad() {
```

Comment:
a) Please don't forget to call `super` on view lifecycle methods
b) General iOS convention is to put view lifecycle methods towards the top of the file, under variable declarations.

Reply:
I'll fix this